### PR TITLE
README for rocAL video unit test

### PR DESCRIPTION
**Pull request by fiona-gladwin**
- Add README.md file for rocAL video unit test.
- Minor fix and changes in rocAL video unit test

### CODE REVIEW

File: utilities/rocAL/rocAL_video_unittests/README.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+The other arguments can be modified in the test script.
+  ````
+./testScript.sh <input> <test-case>
+  ````
+The outputs will be dumped inside the build/output_frames folder
+
+NOTE:
+
+* Inputs for cases 1 and 2 - Video file / folder containing videos
```
Comment (rrawther) [inline utilities/rocAL/rocAL_video_unittests/README.md:29]: we need to explain what testcases 1 and 2 does and show examples with video files and folder

**Review by rrawther — CHANGES_REQUESTED** [review]
It didn't work for me with relative path. Also it hangs for the first test case

**Comment by kiritigowda** [comment]
@fiona-gladwin any updates on this PR?

**Comment by LakshmiKumar23** [comment]
@kiritigowda I have asked Fiona to update the README

**Comment by fiona-gladwin** [comment]
> It didn't work for me with relative path. Also it hangs for the first test case

Hi Rajy, I have added a fix for this issue please check

**Comment by kiritigowda** [comment]
@LakshmiKumar23 can you verify this fix

File: utilities/rocAL/rocAL_video_unittests/README.md [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+> ./testScript.sh <[path/to/test_frame_num.mp4](https://github.com/GPUOpen-ProfessionalCompute-Libraries/MIVisionX-data/blob/main/rocal_data/video_and_sequence_samples/test_frame/test_frame_num.mp4)> 1
+
+Arguments to be modified in testScript.sh to get the following output:
+
+- BATCH_SIZE=2
+- SEQUENCE_LENGTH= 3
+- STEP=3
+- STRIDE=5
+
+![video_reader.png](./samples/video_reader.png)
```
Comment (LakshmiKumar23) [inline utilities/rocAL/rocAL_video_unittests/README.md:95]: @fiona-gladwin can you check this output image? I have
0 5 10
3 8 13
Which makes sense. not sure how you have 14

Comment (LakshmiKumar23) [reply utilities/rocAL/rocAL_video_unittests/README.md:95]: @kiritigowda @rrawther I don't see any hang issues. Just need to cross check output from my previous comment before merging

Comment (fiona-gladwin) [reply utilities/rocAL/rocAL_video_unittests/README.md:95]: @LakshmiKumar23 I have replaced the image with the correct sequence please check

**Review by LakshmiKumar23 — APPROVED** [review]

**Pull request merged**